digiwf-connector/digiwf-connector-starter/src/test/java/de/muenchen/oss/digiwf/connector/core/adapter/in/streaming/BpmnErrorEventConsumerTest.java

```diff
@@ -0,0 +1,49 @@
+package de.muenchen.oss.digiwf.connector.core.adapter.in.streaming;
+
+import de.muenchen.oss.digiwf.connector.core.application.port.in.CreateBpmnErrorInPort;
+import de.muenchen.oss.digiwf.connector.core.domain.BpmnError;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import org.springframework.messaging.Message;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+class BpmnErrorEventConsumerTest {
+
+    private BpmnErrorEventConsumer consumer;
+    private CreateBpmnErrorInPort inPort;
+
+    @BeforeEach
```

Comment:
**Suggestion:** Add a docstring to the 'setUp' method
```suggestion
    @BeforeEach
    void setUp() {
        ...
    }
```

Reply:
Die Suggestion ergibt keinen Sinn